keanu-project/src/main/java/io/improbable/keanu/vertices/Vertex.java

```diff
@@ -224,7 +216,7 @@ public int hashCode() {
         return (int) (uuid ^ (uuid >>> 32));
     }
 
-    public Set<Vertex> getConnectedGraph() {
+    public Set<Vertex<?>> getConnectedGraph() {
```

Comment:
The linter doesn't like this. It's due to it forcing the user to use generics...
https://rules.sonarsource.com/java/RSPEC-1452